src/components/views/rooms/LinkPreviewWidget.js

```diff
@@ -73,36 +73,13 @@ module.exports = React.createClass({
         this.unmounted = true;
     },
 
-    onImageClick: function(ev) {
-        var p = this.state.preview;
-        if (ev.button != 0 || ev.metaKey) return;
-        ev.preventDefault();
-        var ImageView = sdk.getComponent("elements.ImageView");
-
-        var src = p["og:image"];
-        if (src && src.startsWith("mxc://")) {
-            src = MatrixClientPeg.get().mxcUrlToHttp(src);
-        }
-
-        var params = {
-            src: src,
-            width: p["og:image:width"],
-            height: p["og:image:height"],
-            name: p["og:title"] || p["og:description"] || this.props.link,
-            fileSize: p["matrix:image:size"],
-            link: this.props.link,
-        };
-
-        Modal.createDialog(ImageView, params, "mx_Dialog_lightbox");
-    },
-
```

Comment:
It was following on from the discussion of whether it should open the link, or show a preview of the image. If a URL is posted and Vector attaches a preview, a user would expect to click-through to the URL rather than open a lightbox. If the URL is linking to an image type, then perhaps an exception could be made to trigger the lightbox instead. Is the content-type available?